src/generators/php/index.js

```diff
@@ -141,11 +156,22 @@ class PHP extends Generator {
       case "null":
         return "null";
       default:
-        if (enumMap[typeName]) {
-          // TODO: OA and SchemaOrg use same namespace: do we need to differentiate?
+        if (enumMap[compactedTypeName]) {
+          if (this.includedInSchema(enumMap[compactedTypeName].namespace)) {
+            return (
+              "\\OpenActive\\Enums\\SchemaOrg" +
+              this.convertToCamelCase(typeName)
+            );
+          }
           return "\\OpenActive\\Enums\\" + this.convertToCamelCase(typeName);
         } else if (modelsMap[typeName]) {
-          return this.convertToCamelCase(typeName);
+          if (this.includedInSchema(modelsMap[typeName].namespace)) {
+            return (
+              "\\OpenActive\\Models\\SchemaOrg" +
+              this.convertToCamelCase(typeName)
+            );
+          }
+          return "\\OpenActive\\Models\\OA" + this.convertToCamelCase(typeName);
```

Comment:
Same as above:
Missing trailing `\\` so that fully namespaced name is for example `\OpenActive\Models\OA\EntryPoint` instead of `\OpenActive\Models\OAEntryPoint`